python/lsst/analysis/tools/actions/plot/diaSkyPlot.py

```diff
@@ -155,10 +156,12 @@ def _makePanel(self, data, panel, ax, **kwargs):
         ax : matplotlib axis
         color : `str`
         """
-        ras = cast(Vector, data[panel.ra])
-        decs = cast(Vector, data[panel.dec])
+        for ra, dec in zip(panel.ras, panel.decs):  # loop over column names (dict keys)
+            ax.scatter(data[ra], data[dec], s=panel.size, alpha=panel.alpha, marker=".", linewidths=0)
+            # TODO DM-42768: implement lists of colors, sizes, alphas, etc.
+            # and add better support for multi-panel plots.
+            # Right now, color is excluded, each series gets the next default.
```

Comment:
Another suggestion, but probably for a future ticket. There should probably be some sort of 2d binning for dense data